### DISCUSSION TITLE
How to exclude a range of versions?

### DISCUSSION THREAD

**Discussion by pjljvandelaar** [alire-project/alire#1175] [Q&A] [answered]
Hi All,

I would like to exclude a range of versions (to be precise 12.1.0 up to and including 12.1.2)
but I failed.

when I use "/=12.1.1 /= 12.1.2" Alire complaints about the second "/=".
when I use "<=11.2.4 >= 12.1.3" Alire complaints about ">="

Is it not possible, can one only specify below, above and one exception?

Greetings,
   Pierre

**Comment by Fabien-Chouteau** [comment, marked as answer]
From the Alire doc, but difficult to find ^^ :

> Logical operators for and (&), or (|) are accepted; see the Semantic_Versioning project documentation on [extended version sets](https://github.com/alire-project/semantic_versioning#types).

@mosteo I think we need a dedicated doc section on versioning.

**Reply by mosteo** [reply]
Created https://github.com/alire-project/alire/issues/1187 to track this one.
